src/energyplus/ForwardTranslator/ForwardTranslateAirTerminalSingleDuctInletSideMixer.cpp

```diff
@@ -108,6 +114,36 @@ namespace energyplus {
       idfObject.setString(AirTerminal_SingleDuct_MixerFields::MixerSecondaryAirInletNodeName, secondaryInletNode->name().get());
     }
 
+    // ControlForOutdoorAir
+    {
+      if (modelObject.controlForOutdoorAir()) {
+        bool dsoa_found = false;
+        if (auto airLoopHVAC = modelObject.airLoopHVAC()) {
+          auto zones = airLoopHVAC->demandComponents(modelObject, airLoopHVAC->demandOutletNode(), model::ThermalZone::iddObjectType());
```

Comment:
Editorial: Sure seems like after all this time, we should have a method to get a terminal's zone. Trouble is that there is no Terminal base class.